### ISSUE TITLE
invalid value encountered in divide

### ISSUE THREAD

**Issue by tresacton** [newaetech/chipwhisperer-lint#1]
Hi Colin,

First - thanks for the tool! 

It seems that some tests result in the following error. 

```
/opt/cwlint/cwlint_git/server-backend/server/analysis.py:164: RuntimeWarning: invalid value encountered in divide
  beta = (n*S_xy - S_x*S_y) / (n*S_xx - S_x**2)

```

Some things to note about my environment, in case any of it is useful to you:

- I'm running this on a fresh ubuntu on EC2
- I had to downgrade numpy to numpy-1.16.2 to avoid a pickle=False error
- I had to upgrade gevent to latest (gevent-1.4.0) to avoid a `TypeError: child watchers are only available on the default loop` error
- I get warnings about QT but it's not in the requirements so I'm ignoring it
